reconcile/external_resources/secrets_sync.py

```diff
@@ -120,8 +120,8 @@ def _format_key(self, key: str) -> str:
     def _format_value(self, value: str) -> str:
         decoded_value = base64.b64decode(value).decode("utf-8")
         if decoded_value.startswith("__vault__:"):
-            _secret_ref = json.loads(decoded_value.replace("__vault__:", ""))
-            secret_ref = VaultSecret(**_secret_ref)
+            secret_ref_ = json.loads(decoded_value.replace("__vault__:", ""))
```

Comment:
do you know what's the rationale of this? It just looks a personal preference of somebody 😂

Reply:
Jupp, it's just a strange [convention](https://docs.astral.sh/ruff/rules/used-dummy-variable/)